packages-exp/app-exp/src/api.ts

```diff
@@ -285,7 +285,8 @@ export function onLog(
 ): void {
   if (logCallback !== null && typeof logCallback !== 'function') {
     throw ERROR_FACTORY.create(AppError.INVALID_LOG_ARGUMENT, {
-      appName: name
+      // appName isn't used in INVALID_LOG_ARGUMENT error but ErrorFactory requires it
+      appName: ''
```

Comment:
The correct fix should be change `type ErrorParams` in `errors.ts` to

```
type ErrorParams = {
  [AppError.NO_APP]: { appName: string },
  [AppError.BAD_APP_NAME]: { appName: string },
  [AppError.DUPLICATE_APP]: { appName: string },
  [AppError.APP_DELETED]: { appName: string },
  [AppError.INVALID_APP_ARGUMENT]: { appName: string }
};
```

Reply:
Done.